### PR TITLE
Matrix routing

### PR DESCRIPTION
**Pull request by me-khumoyun**
Pull request for matrix routing with gpt apit

### CODE REVIEW

**Comment by ImAli0** [comment]
nice job, but add comments pls

File: matrix.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+                time = times[i*len(locations) + j]
+                results.append(f"From {locations[i]} to {locations[j]}: {distance} miles, {time//3600} hours {time%3600//60} minutes")
+    return "\n".join(results)
+
+
+# input("\nPlease type cities: ")
+locations = ["New York", "Los Angeles", "Chicago"]
+api_key = "myzf3oUjUSCk3Lhn0nfgSHqB4DU4v4CM"
+result = get_distances_and_times(locations, api_key)
+print(result)
```
Comment (ImAli0) [inline matrix.py:37]: delete unused prints

**Review by ImAli0 — APPROVED** [review]
Please keep clean code

**Comment by me-khumoyun** [comment]
done

**Pull request merged**